subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/GradleVsMavenPerformanceTestRunner.groovy

```diff
@@ -69,7 +69,6 @@ class GradleVsMavenPerformanceTestRunner extends AbstractGradleBuildPerformanceT
             invocationCount = runs
             projectName(testProject).displayName("Maven $commonBaseDisplayName").invocation {
                 tasksToRun(equivalentMavenTasks.split(' ')).mavenOpts(jvmOpts.collect {it.toString()}).args(mvnArgs.collect {it.toString()})
-                    .args('-q', '-Dsurefire.printSummary=false')
```

Comment:
FTR this didn't have any effect on the test results. Spooling the output to a file is blazing fast.